lib/basedriver/commands/session.js

```diff
@@ -19,7 +19,14 @@ commands.createSession = async function (caps) {
   // merge caps onto opts so we don't need to worry about what's where
   Object.assign(this.opts, this.caps);
 
-  // deal with reset
+  // deal with resets
+  // some people like to do weird things by setting noReset and fullReset
+  // both to true, but this is misguided and strange, so error here instead
+  if (this.opts.noReset && this.opts.fullReset) {
+    throw new Error("The `noReset` and `fullReset` capabilities are mutually " +
```

Comment:
Elsewhere, generally, the quotes that get into the logs are single normal quotes (`'`), rather than backticks.